commands/dictionary.js

```diff
@@ -0,0 +1,79 @@
+var GReYBot = require('../greybot');
+var util = require('util');
+var xml2js = require('xml2js');
+var parser = new xml2js.Parser();
+
+exports.commands = [
+	'define'
+];
+
+exports.define = {
+	usage: '<word>',
+	description: 'looks up a word in the Merriam-Webster Collegiate Dictionary',
+	process: (msg, suffix) => {
+		Array.prototype.clean = function(deleteValue) {
+			for (var i = 0; i < this.length; i++) {
+				if (this[i] == deleteValue) {
+				this.splice(i, 1);
+				i--;
+				}
+			}
+			return this;
+		};
+    	var word = suffix;
+		if (!word) {
+			msg.channel.send('I won\'t define an empty string.');
+			return;
+		}
+		var url = 'http://www.dictionaryapi.com/api/v1/references/collegiate/xml/' + word + '?key=' + GReYBot.Auth.dictionary_api_key;
+		require('request')(url,
```

Comment:
valid point, but we only do that in instances where something is used more than once. all the other plugins follow this convention